qa-test-apps/overrideTargets/nginx.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-override-test
+  labels:
+    app: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.14.2
```

nit: nginx has [more recent images](https://hub.docker.com/_/nginx) by now ;)
```suggestion
        image: nginx:1.29
```